lang.json

```
@@ -32,374 +37,447 @@
 			"en": "This will be the default layout for your sketches",
-			"ja": "モード"
+			"ja": "モード",
+			"pt": "Este será a visualização padrão para seus esboços"
 		},
 		"console": {
 			"description": "code settings > editor",
 			"en": "Console",
-			"ja": "モード"
+			"ja": "モード",
+			"pt": "Console"
 		},
 		"consoleDescription": {
 			"description": "code settings > editor",
 			"en": "It will show up when there is an error or print() in code",
-			"ja": "モード"
+			"ja": "モード",
+			"pt": "Será exibido quando houver um erro ou print() no código"
 		},
 		"linting": {
 			"description": "code settings > editor",
 			"en": "Linting",
-			"ja": "モード"
+			"ja": "モード",
+			"pt": "Linting"
 		},
 		"lintingDescription": {
 			"description": "code settings > editor",
 			"en": "Potential warnings will be displayed as you type",
-			"ja": "モード"
+			"ja": "モード",
+			"pt": "Avisos em potencial serão exibidos conforme você digita"
 		},
 		"lintingDescription-p5js": {
 			"description": "code settings > editor. Shows when sketch is not p5js.",
 			"en": "Disabled: Only available on p5js sketches.",
-			"ja": "モード"
+			"ja": "モード",
+			"pt": "Desabilitado: Disponível somente nos esboços de p5js."
 		},
 		"shortcuts": {
 			"description": "code settings > editor",
 			"en": "Shortcuts",
-			"ja": "モード"
+			"ja": "モード",
+			"pt": "Atalhos"
 		},
 		"seeMoreShortcuts": {
 			"description": "code settings > editor",
 			"en": "Shortcuts",
-			"ja": "モード"
+			"ja": "モード",
+			"pt": "Ver mais atalhos"
 		},
 		"versionsSummarized": {
 			"description": "code settings > versions. Displayed when sketch has more than 20 versions.",
 			"en": "Prior to most recent 20 versions are summarized.",
-			"ja": "モード"
+			"ja": "モード",
+			"pt": "Prévia das 20 versões mais recentes são resumidas."
 		},
 		"plusPromo": {
 			"description": "code settings > promo. Displayed at the bottom if user is not a Plus+ member",
 			"en": "{0} to add custom libraries, private sketches, and more!",
-			"ja": "{0} モード"
+			"ja": "{0} モード",
+			"pt": "{0} adicionar mais bibliotecas personalizadas, esboços privados, e mais!"
 		}
 	},
 	"user": {
 		"name": {
 			"description": "Edit Profile label",
 			"en": "name",
-			"ja": "モード"
+			"ja": "モード",
+			"pt": "nome"
 		},
 		"location": {
 			"description": "Edit Profile > label",
 			"en": "location",
-			"ja": "モード"
+			"ja": "モード",
+			"pt": "localização"
 		},
 		"website": {
 			"description": "Edit Profile > label",
 			"en": "website",
-			"ja": "モード"
+			"ja": "モード",
+			"pt": "website"
 		},
 		"email": {
 			"description": "Edit Profile > label",
 			"en": "email",
-			"ja": "モード"
+			"ja": "モード",
+			"pt": "email"
 		},
 		"password": {
 			"description": "Edit Profile > label",
 			"en": "password",
-			"ja": "モード"
+			"ja": "モード",
+			"pt": "senha"
 		},
 		"clickToChange": {
 			"description": "Edit Profile > password change link",
 			"en": "Click to Change",
-			"ja": "モード"
+			"ja": "モード",
+			"pt": "Clique para mudar"
```

```suggestion
			"pt": "Clique para Mudar"
```
Capitalizing last word?